src/pages/SearchList/LocationModal.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import styled from "styled-components";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faHotel } from "@fortawesome/free-solid-svg-icons";
+
+const LocationModal = ({ locationInput }) => {
+  const sortedList = LOCATION_DATA.filter(hotels => {
+    return hotels.location.includes(locationInput);
+  });
+
+  return (
+    <LocationBox>
+      <LocationListContainer>
+        <HotelList lists={sortedList} />
+      </LocationListContainer>
+    </LocationBox>
+  );
+};
+
+const HotelList = ({ lists }) => {
+  return lists.map(list => {
+    return (
+      <LocationListBox key={lists.id}>
+        <HotelIcon icon={faHotel} size="2x" />
+        <div>
+          <HotelName>{list.hotel_name}</HotelName>
+          <HotelLocation>{list.location}</HotelLocation>
+        </div>
+      </LocationListBox>
+    );
+  });
+};
```

Comment:
하나의 파일에서 하나의 컴포넌트만 작성해서 각 파일이 독립적인 컴포넌트를 가질 수 있도록 해주세요!
그리고 이 경우 다음과 같이 합치시면 될 것 같네요!
```suggestion
const LocationModal = ({ locationInput }) => {
  const sortedList = LOCATION_DATA.filter(hotels => {
    return hotels.location.includes(locationInput);
  });

  return (
    <LocationBox>
      <LocationListContainer>
        {sortedList.map(list => {
          return (
            <LocationListBox key={lists.id}>
              <HotelIcon icon={faHotel} size="2x" />
              <div>
                <HotelName>{list.hotel_name}</HotelName>
                <HotelLocation>{list.location}</HotelLocation>
              </div>
            </LocationListBox>
          );
        })}
      </LocationListContainer>
    </LocationBox>
  );
};
```

Reply:
수정 완료

Reply:
수정 완료